bin/java-petstore-native.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+SCRIPT="$0"
+echo "# START SCRIPT: $SCRIPT"
```

Comment:
please also add a windows script under `bin/windows/`

Reply:
Done. Thanks for catching that!